src/metabase/api/dashboard.clj

```diff
@@ -96,6 +97,7 @@
                 (db/insert! Dashboard dashboard-data))]
       ;; publish event after the txn so that lookup can succeed
       (events/publish-event! :dashboard-create dash)
+      (analytics/track-event :dashboard_created api/*current-user-id* {:dashboard_id (u/the-id dash)})
```

Comment:
Is there any situation where we'd want to pass a `user-id` other than `*current-user-id*`? Should `*current-user-id*` be the default? It seems to me like `track-event` could just grab `*current-user-id*` directly instead of requiring each call to explicitly pass it in

Reply:
During the setup API call, we need to pass the user ID of the new admin who was just created, which won't (yet) be `*current-user-id*`.